test/regressions/index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html>
+  <head>
+    <title>vrtest</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <style>
+      body {
+        background-color: white;
+      }
+    </style>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <script>
+      var fakeNow = new Date('Mon Aug 18 14:11:54 2014 -0500');
```

Comment:
Mock date the same/similar way as in e2e tests.
Sinon fake timers no longer work on Vite. 🙈